lib/vector-config/src/configurable.rs

```diff
@@ -1,3 +1,8 @@
+#![deny(missing_docs)]
```

Comment:
Seems like this should be in `src/lib.rs`?

Reply:
It should be, yes, but that causes >60 missing docs errors that I didn't want to deal with.

Reply:
Ohhh, gotcha. Fair, fair. :)